.github/workflows/propose_semver_release.yml

```diff
@@ -0,0 +1,125 @@
+name: Propose SemVer Release
+on:
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+        default: dev
+      action_branch:
+        description: 'The shared action branch to checkout'
+        type: string
+        default: main
+      python_version:
+        description: 'The python version to use'
+        type: string
+        default: "3.8"
+      version_file:
+        description: 'The file location to read the version from'
+        type: string
+        default: version.py
+      release_type:
+        type: string
+        description: 'The type of release to make'
+        required: true
+      changelog_file:
+        type: string
+        default: CHANGELOG.md
+      
+
+jobs:
+  bump_version:
+    runs-on: ubuntu-latest
+    env:
+      VERSION_FILE: ${{ github.workspace }}/action/package/${{ inputs.version_file }}
+      TARGET_BRANCH: ${{ inputs.release_type == 'patch' && 'testing' || 'stable' }}
+      GITHUB_REPOSITORY: ${{ github.repository }}
+      RELEASE_TYPE: ${{ inputs.release_type }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.branch }}
+          path: action/package/
+      - name: Checkout Scripts Repo
+        uses: actions/checkout@v4
+        with:
+          path: action/github/
+          ref: ${{ inputs.action_branch }}
+          repository: OpenVoiceOS/.github
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ inputs.python_version }}
+      - name: Install Python Dependencies
+        run: |
+          pip install -r ${{ github.workspace }}/action/github/requirements/pip_base.txt
```

Comment:
[No such file or directory: '/home/runner/work/skill-ovos-randomness/skill-ovos-randomness/action/github/requirements/pip_base.txt'](https://github.com/OpenVoiceOS/skill-ovos-randomness/actions/runs/7866728052/job/21461392522)

Reply:
Is `inputs.action_branch` defined in your call? Looks like you'll need to pass this branch to the workflow call as an input

Reply:
Don't bother with release workflows at all.

Like we have established, the workflows have to be rewritten to fit the new flow.
As this is WIP anyhow i don't suggest adopting it to a productive environment.
inputs and names in general will change.

The `action_branch` is needed as the correct (script) branch has to be checked out (we are working on a feature branch after all)